nimbix_desktop/nimbix_desktop

```diff
@@ -129,7 +130,7 @@ fi
 tools=/usr/lib/JARVICE/tools
 #[[ -f /etc/init.d/messagebus ]] && SVC=messagebus || SVC=dbus
 #service $SVC start
-#DISPLAY=:1 dbus-launch
+DISPLAY=:1 dbus-launch
 
```

Comment:
What is needed to remove dbus requirement

Reply:
Its removed 😊